src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,72 @@
+package lambda.part1.exercise;
+
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise3 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort + expression-lambda
+        Arrays.sort(persons, (p1, p2) -> {
+            return Integer.compare(p1.getAge(), p2.getAge());
```

Comment:
I'll be glad if you change this statement-lambda to expression-lambda, as this exercise requires